api/swagger-spec/v1beta1.json

```diff
@@ -2289,11 +2289,11 @@
      },
      "metadata": {
       "$ref": "v1.ObjectMeta",
-      "description": "Standard object metadata. More info: http://releases.k8s.io/HEAD/docs/devel/api-conventions.md#metadata"
+      "description": "Standard object metadata. More info: http://releases.k8s.io/v1.1.0/docs/devel/api-conventions.md#metadata"
```

Comment:
@nikhiljindal @RichieEscarez , is it v1.1.0 or v1.1? What URL will put our docs in k8s.io under? Thanks.